### DISCUSSION TITLE
Welcome to Noi Discussions!

### DISCUSSION THREAD

**Discussion by lencx** [lencx/Noi#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Alster69** [comment]
how is the privacy of this app, can any body tell if u can login using google and if conversations are recorded?

**Reply by lencx** [reply]
It is a localized application, Noi does not have servers running, and the data you generate will be stored locally on your system. The only place that might provide data configuration or plugins is this repository, which is open source.

**Comment by Fabiano1871** [comment]
Hey there,

I’ve been using Noi on Debian, and I’m really happy with it! I’m about to swap out some hardware, including my SSD, so I’ll be doing a clean install of the system.

I’m backing up all my apps along with their config files, but I’m not sure which file in Noi stores the credentials. I don’t need it for anything shady—just want to get everything up and running quickly without having to log in to every AI service again.

Thanks a lot!

**Comment by JairPPereira** [comment]
Desde ontem o ChatGPT está apresentando problema no NOI, fica todo tempo recarregando a pagina e assim não está sendo possivel usar o ChatGPT, outras IAs estão funcionando sem problema

[a comment by an automated account is not shown]
